go.mod

```diff
@@ -208,3 +208,5 @@ replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-d
 go 1.21.4
 
 retract v0.0.2
+
+replace github.com/lightninglabs/neutrino => github.com/Chinwendu20/neutrino v0.0.0-20240514174050-3275e33e5504
```

Comment:
can point to the merge commit instead now so that the ci can run against latest version 👍